calls/forms.py

```diff
@@ -14,6 +19,27 @@
 CALLS_MAX_DISTANCE_MILES = settings.CALLS_MAX_DISTANCE_MILES
 CALLS_MAX_LIST_SIZE = settings.CALLS_MAX_LIST_SIZE
 
+class CommaSeparatedCallersTextArea(Widget):
+    """
+    Transform many-to-many callers into comma separated values for display.
+    """
+    def render(self, name, value, attrs=None, renderer=None):
+        final_attrs = self.build_attrs(attrs, {'type':'text', 'name':name})
+        caller_emails = []
+
+        if value is not None:
+            """Loop through the caller_ids we have stored on the many-to-many
+            relationship and get the corresponding object"""
+            for caller_id in value:
+                callprofile = CallProfile.objects.get(pk=caller_id)
```

Comment:
I'm not sure if there should be any db queries happening in a form widget render method, but at least it's just a get and not data change.

Reply:
I agree, I'm going to try and write out the widget.